### PR TITLE
feat: Modal component

### PR DESCRIPTION
**Pull request by Sameony**
Added modal component
image

### CODE REVIEW

**Review by shuntzhere — APPROVED** [review]
No change is required. Looks good

File: Templates/Modal/modal.html [hunk 1]
```diff
@@ -168,7 +168,41 @@ <h1 class="pd-t-md"> Mirinda<span class="prim-color">UI</span> </h1>
...
+                Are you sure you want to follow that person who gave you chocolate?
+                <div class="flex flex-spc-arnd">
+                    <button class="btn mg-sm btn-success">Yes</button>
+                    <button class="btn mg-sm btn-caution">No </button>
+                </div>
+            </div>
+        </div>
+            </code>
+        </pre>
+    </div>
```
Comment (vaibhav-mahalle) [inline Templates/Modal/modal.html:205]: looks good no changes required.

**Review by vaibhav-mahalle — APPROVED** [review]

File: Templates/Modal/modal.html [hunk 2]
```diff
@@ -168,7 +168,41 @@ <h1 class="pd-t-md"> Mirinda<span class="prim-color">UI</span> </h1>
...
+                </div>
+            </div>
+        </div>
+        <pre>
+            <code class="showcase">
+        <div class="modal">
+            <h2>Confirmation</h2>
+            <div class="contentArticle pd-sm">
+                Are you sure you want to follow that person who gave you chocolate?
+                <div class="flex flex-spc-arnd">
```
Comment (Asypher00) [inline Templates/Modal/modal.html:197]: i think you can try using class names different from css properties.

File: Templates/Modal/modal.html (hunk 1 above)
Comment (Asypher00) [inline Templates/Modal/modal.html:205]: The code seems very neat and readable. Just suggested one change.

**Review by Asypher00 — COMMENTED** [review]
The code seems very neat and readable. Just suggested one change.

**Pull request merged**